mappings/net/minecraft/block/entity/BlockEntityType.mapping

```diff
@@ -17,7 +17,12 @@ CLASS net/minecraft/class_2591 net/minecraft/block/entity/BlockEntityType
 		FIELD field_11915 supplier Lnet/minecraft/class_2591$class_5559;
 		FIELD field_19316 blocks Ljava/util/Set;
 		METHOD <init> (Lnet/minecraft/class_2591$class_5559;Ljava/util/Set;)V
+			ARG 1 creator
 			ARG 2 blocks
 		METHOD method_11034 build (Lcom/mojang/datafixers/types/Type;)Lnet/minecraft/class_2591;
 		METHOD method_20528 create (Lnet/minecraft/class_2591$class_5559;[Lnet/minecraft/class_2248;)Lnet/minecraft/class_2591$class_2592;
 			ARG 1 blocks
+	CLASS class_5559 BlockEntityCreator
```

Comment:
`BlockEntityFactory`?

Reply:
Yeah. that's much better. (And actually follows the conventions :p )